src/test/CMakeLists.txt

```diff
@@ -628,7 +628,7 @@ if(PROMTOOL_EXECUTABLE)
 else()
   if(LINUX)
     add_ceph_test(run-promtool-unittests
-      {CMAKE_CURRENT_SOURCE_DIR}/run-promtool-unittests.sh)
+      ${CMAKE_CURRENT_SOURCE_DIR}/run-promtool-unittests.sh)
```

Comment:
could you revise the title like

> cmake: add the missing $ prefix in cmake variable
>
> otherwise `run-promtool-unittests` fails to run if `promtool` is not available

Reply:
actually, i think we should drop the `run-promtool-unittests.sh` script. @aaSharma14 what do you think?

Reply:
I'm for ripping it out, and backporting as far as needed, since it is in all version back as far as nautilus.

Reply:
@tchaikov . The purpose of this script is to run the promtool unit tests explicitly if required( I mean without running the whole make check). But this can also  be done by simply running this command `promtool test rules test_alerts.yml` in this `monitoring/prometheus/alerts` directory. So IMHO we can remove the script, but maybe we should document how to run these tests explicitly in readme or dash_devel.

Reply:
@aaSharma14 do you mind creating a PR to implement this ?

Reply:
Yes Sure, will do that.

Reply:
@aaSharma14 
Have not seen a PR te remove the script reference.
And the bug is still there.

Reply:
Submitted these 2 PR's for the same. @tchaikov @wjwithagen PTAL -
https://github.com/rhcs-dashboard/ceph-dev/pull/39
https://github.com/ceph/ceph/pull/41597